server/routes/link/VerifyLinkController.ts

```diff
@@ -35,8 +35,9 @@ export default class VerifyLinkController {
         req.flash('errors', [
           { href: '#email', text: 'The link you used is no longer valid. Request a new one to sign in.' },
         ])
-        res.redirect('/link/request-link')
+        return res.redirect('/link/request-link')
```

Comment:
do we also need this on line 28? bit confused as to how this ever worked!

Reply:
I'll explain it tomorrow, but basically it worked by luck by virtue of the fact that the error message rendered in the flash object for both use cases is the same (lines 26 and 36).
If they had different error messages we would have noticed this bug sooner.
The only reason I noticed it here is that `res.render` was being called twice (I'll explain why tomorrow) and the `supertest` http client (`superagent`) complained bitterly that the response had already been written to and that it couldnt do it again!